compiler/rustc_feature/src/active.rs

```diff
@@ -487,6 +487,10 @@ declare_features! (
     (incomplete, repr128, "1.16.0", Some(56071), None),
     /// Allows `repr(simd)` and importing the various simd intrinsics.
     (active, repr_simd, "1.4.0", Some(27731), None),
```

Comment:
This is a "perma-unstable" feature, just for testing purposes...at least thus far...so can you move it up to the group above, e.g. where `rustc_private` and so forth are found? And maybe rename to `rustc_strict_coherence`?